nipype/external/six.py

```diff
@@ -186,8 +187,8 @@ def find_module(self, fullname, path=None):
     def __get_module(self, fullname):
         try:
             return self.known_modules[fullname]
-        except KeyError:
-            raise ImportError("This loader does not know module " + fullname)
+        except KeyError as e:
+            raise_from(ImportError("This loader does not know module " + fullname), e)
```

Comment:
we should update external files before a release, but any changes should really be upstream. six and future are competing packages and at present we are using both. it will require a focused sprint to use one or the other.